iron-menu-behavior.html

```diff
@@ -208,24 +208,50 @@
     },
 
     /**
+     * Returns the item in the first menu from where the event was dispatched.
+     *
+     * @param {Event}
+     */
+    _isClosestNode: function(event) {
```

Comment:
This method name could probably use some work. `is` suggests you are comparing the type or identity of the object passed as the first argument. `ClosestNode` suggests you are comparing values, and otherwise looking for the closest of something more general than a "menu".